src/index.js

```diff
@@ -12,6 +12,14 @@ const createElmMiddleware = (elm) => {
     }
   const run = store => {
     window.__REDUX_ELM_STORE__ = store;
+    if (elm && elm.ports && elm.ports.elmToRedux) {
+      elm.ports.elmToRedux.subscribe(payload => {
+        store.dispatch({
+          type: '@@elm/json',
+          payload
```

Comment:
Great idea. I'll look into this. I'm not exactly sure how to do this (that is why I used a generic name).